src/codegen/parser.cpp

```diff
@@ -1029,13 +1030,6 @@ AST_Module* parse_file(const char* fn, FutureFlags inherited_flags) {
     return rtn;
 }
 
-const char* getMagic() {
-    if (ENABLE_PYPA_PARSER)
-        return "a\ncR";
-    else
-        return "a\nCR";
-}
-
 #define MAGIC_STRING_LENGTH 4
```

Comment:
could you please change this to
`const int MAGIC_STRING_LENGTH = 4` and move it right beneath `const char* MAGIC = "a\nCR";`